Configuration.props

```diff
@@ -56,6 +56,9 @@
     <AndroidMxeInstallPrefix Condition=" '$(HostOS)' == 'Linux' ">\usr</AndroidMxeInstallPrefix>
     <AndroidSdkDirectory Condition=" '$(AndroidSdkDirectory)' == '' ">$(AndroidToolchainDirectory)\sdk</AndroidSdkDirectory>
     <AndroidNdkDirectory Condition=" '$(AndroidNdkDirectory)' == '' ">$(AndroidToolchainDirectory)\ndk</AndroidNdkDirectory>
+    <AndroidSdkCmakeDirectory>$(AndroidSdkDirectory)\cmake\3.6.4111459</AndroidSdkCmakeDirectory>
+    <CmakePath Condition=" '$(CmakePath)' == '' ">$(AndroidSdkCmakeDirectory)\bin\cmake</CmakePath>
+    <NinjaPath Condition=" '$(NinjaPath)' == '' ">$(AndroidSdkCmakeDirectory)\bin\ninja</NinjaPath>
```

Comment:
Should those CmakePath and NinjaPath be public properties and made customizible?

We historically had those tools customizible, and often brought in problems that they were not actually customizible. We don't write tests for customizibility because writing tests means doubling test run time.

If we just need to bring customizability as workaround for possible future incompatibility, I'd make them implicitly non-public (i.e. '_'-prefixed).

Reply:
It doesn't hurt if someone wants to use a different cmake or ninja - we default to the version from the SDK since that's convenient for us, but the only thing we rely on is the NDK's Android toolchain definition otherwise.